client/filejsonstore/filejsonstore_test.go

```diff
@@ -0,0 +1,175 @@
+package client
+
+import (
+	"encoding/json"
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"gopkg.in/check.v1"
+
+	"github.com/theupdateframework/go-tuf/client"
+)
+
+type RawJSONStoreSuite struct{}
```

Comment:
Can we name these tests consistently? (`RawJSONStoreSuite -> FileJSONStoreSuite`)

Reply:
👍